src/components/UserActions/Strings.ts

```diff
@@ -23,6 +23,8 @@ Translation.register(Language.English, {
     UserActivity_duration: 'Duration',
     UserActivity_other_event: 'Other Event',
     UserActivity_other_events: 'Other Events',
+    UserActivity_no_actions_timeline: 'No actions to display in the timeline',
```

Comment:
Potential causes-->Possible causes

Reply:
Sure, that will change it for the other places where the error is displayed is that okay?